src/connectors/menu/__tests__/connectMenu-test.js

```diff
@@ -230,6 +230,59 @@ describe('connectMenu', () => {
     ]);
   });
 
+  it('provides the correct transformed facet values', () => {
+    const widget = makeWidget({
+      attributeName: 'category',
+      transformItems: items =>
+        items.map(item => ({
+          ...item,
+          label: 'transformed',
+        })),
+    });
+
+    const helper = jsHelper({}, '', widget.getConfiguration({}));
+    helper.search = sinon.stub();
```

Comment:
When we touch a test file that is still using sinon, we try to take the opportunity to replace it with Jest mocks. Can you do that? 🙏

Reply:
For sure! I initially didn't want to edit too much code as it was intended to be merged on the yet-to-be consequent `feat/3.0` branch.